integrations/Appcues/browser.js

```diff
@@ -0,0 +1,77 @@
+import logger from "../../utils/logUtil";
+import ScriptLoader from "../ScriptLoader";
+
+class Appcues {
+  constructor(config) {
+    this.accountId = config.accountId;
+    this.apiKey = config.apiKey;
+    this.name = "APPCUES";
+    //this.sendToAllDestinations = config.sendToAll;
+  }
+
+  init() {
+    logger.debug("===in init Appcues===");
+    ScriptLoader(
+      "appcues-id",
+      `https://fast.appcues.com/${this.accountId}.js`
+    );
+  }
+
+  isLoaded() {
+    logger.debug("in appcues isLoaded");
+    return !!window.Appcues;
+  }
+
+  isReady() {
+    logger.debug("in appcues isReady");
+    // if (this.sendToAllDestinations) {
```

Comment:
is this needed?

Reply:
We are still discussing regarding it. So commented it for now ,might use it in future if needed

Reply:
ok sure... comment something like // TODO too